counsel.el

```diff
@@ -4107,7 +4123,10 @@ candidate."
                         level
                         todo
                         (if priority (format "[#%c]" priority))
```

Comment:
Ditto re: preferring `and` over conditionals.

Reply:
Why not replace the surrounding `(cl-remove-if 'null ...)` with `(delq nil ...)` while we're at it?